.github/workflows/update_template.yml

```diff
@@ -1,11 +1,9 @@
 name: Update Template
 
 on:
-  workflow_run:
-    workflows: [Release]
-    types:
-      - completed
-  workflow_dispatch:
+  push:
+    tags:
+      - create-svelte@*
```

Comment:
Are we actually tagging? I thought we weren't? This will never run if we aren't.

Reply:
That's the other part of this PR. According to changesets docs, they always tag on publish, but unless you push those tags, they get discarded.

Reply:
Are we going to start publishing with changesets then instead of doing it manually?

Reply:
we already do publish with changesets, no?